packages/replayio/src/utils/date.ts

```diff
@@ -0,0 +1,47 @@
+import differenceInCalendarDays from "date-fns/differenceInCalendarDays";
+import differenceInMinutes from "date-fns/differenceInMinutes";
+import differenceInMonths from "date-fns/differenceInMonths";
+import differenceInWeeks from "date-fns/differenceInWeeks";
+import differenceInYears from "date-fns/differenceInYears";
+import padStart from "lodash.padstart";
```

Comment:
native `''.padStart` is supported since node v8 :)

Reply:
Good point!